src/main/groovy/com/github/jrubygradle/internal/GemVersionResolver.groovy

```diff
@@ -4,10 +4,52 @@ import org.gradle.api.artifacts.DependencyResolveDetails
 
 class GemVersionResolver {
 
-    private Map versions = [:]
-  
+    Map versions
+    private final Object configuration
+    private final Object logger
+
+    static void setup(Object project) {
+        project.configurations.each {
```

Comment:
hoops - this needed a lot of reading between the words - sorry, next time I will read a second time before submitting